reference_guide/custom_language_support/declarations_and_references.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Declarations and References
+---
+<!-- Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+> **WARNING** This API is available starting from 2020.3 and currently in development and thus in experimental state.
+
+## Declarations
+
+Each [symbol](symbols.md) may be declared in zero or more places, for example:
+- JVM package is a symbol with several declarations (split packages);
+- C# partial class is a symbol with several declarations;
```

Comment:
Why backticks? "partial" is not a keyword here

Reply:
It's a judgement call for you as the author. I suggest making "partial" a keyword because it has special significance in the C# language. In this context the word doesn't mean generally incomplete or biased, but has must meet a specific set of criteria in the language.

Reply:
I'll leave it without backticks because here I'm referring to a language feature name as in: "_Partial classes_ are classes marked with `partial` modifier".